### PR TITLE
fav feature added

### LINKED ISSUE #81: Add a Favorite feature
### Suggestion / Feature Request

Add a feature to favorite a movie or tv show. The favorite icon should appear in the detail fragments. The favorites can be stored as a field of type `MutableList<ListItem>` in the `User` data class

### PR DESCRIPTION
**Pull request by armaanbadhan**
## Description

now you can mark your fav movies and tv shows

## Issue Resolved

closes #81 

## Screenshots 

<div>

</div>

## Checklist

Please make sure to review the following before submitting your PR: 

- [x] I have read the [contribution guidelines](https://github.com/PEC-CSS/MovieDroid/blob/main/CONTRIBUTING.md).
- [x] I have read the code of conduct.
- [x] I have reviewed my submission in detail.

### CODE REVIEW

File: app/src/main/java/com/pec_acm/moviedroid/firebase/ListItem.kt [hunk 1]
```diff
@@ -1,10 +1,12 @@
 package com.pec_acm.moviedroid.firebase
 
+import com.pec_acm.moviedroid.model.MovieDetail
 import com.pec_acm.moviedroid.model.MovieResult
+import com.pec_acm.moviedroid.model.TVDetail
 import com.pec_acm.moviedroid.model.TvResult
 
 data class ListItem(
-    val id : Int=0,
+    var id : Int=0,
```
Comment (ken1000minus7) [inline app/src/main/java/com/pec_acm/moviedroid/firebase/ListItem.kt]: Change this back to val

**Review by ken1000minus7 — COMMENTED** [review]
Seems fine, small change needed though, also is this app

**Comment by ken1000minus7** [comment]
Is the fav icon appearing fine on toolbar when collapsed?

**Comment by armaanbadhan** [comment]
yessir

**Review by ken1000minus7 — APPROVED** [review]
Ah nice gw

**Pull request merged**